ReactNative/ReactTwitter/core/views/tweetsList.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react'
+import { ListView } from 'react-native'
+import TweetsListItem from './tweetsListItem'
+
+class TweetsList extends Component {
+
+  constructor (props) {
+    super(props)
+    console.log('constructor')
+    var ds = new ListView.DataSource({rowHasChanged: (r1, r2) => r1 !== r2})
+    this.state = {
+      dataSource: ds.cloneWithRows([])
+    }
+  }
+
+  componentDidMount () {
+    console.log('componentDIdMount')
+    this.refreshData()
+    console.log(this.state)
+  }
+
+  refreshData () {
+    this.setState({
+      dataSource: this.state.dataSource.cloneWithRows([
```

Comment:
👎  It doesn't look more readable as you suggested, it actually looks worse to me.